simpeg_drivers/utils/utils.py

```diff
@@ -129,7 +130,7 @@ def calculate_2D_trend(
 
     params, _, _, _ = np.linalg.lstsq(polynomial, values, rcond=None)
     data_trend = np.zeros(points.shape[0])
-    for count, (x, y) in enumerate(zip(xx, yy)):
+    for count, (x, y) in enumerate(zip(xx, yy, strict=False)):
```

Comment:
**question:** use `strict = True`?

@domfournier or @benk-mira  ?

Reply:
You're repeating yourself bra